### PR TITLE
Added a note for the email address update ff

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pbarapatre10 — DISMISSED** [review]

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ This topic describes how to build a SCIM endpoint using OKTA and integrate it wi
...
 
-### Review: Harness Okta SCIM Integration
+### Review: Harness Okta SCIM integration
 
 By using Okta as your identity provider, you can efficiently provision and manage users in your Harness Account, Org and Project. Harness' [SCIM](https://www.okta.com/blog/2017/01/what-is-scim/) integration enables Okta to serve as a single identity manager, for adding and removing users, and for provisioning User Groups. This is especially efficient for managing many users.
 
 In exchange for the convenience of Okta-provisioned users and groups, you must configure several aspects of Okta, as described in the following sections. You will also have restrictions on modifying Okta-provisioned users and groups natively within Harness, as described in [Limitations](#limitations).
 
-#### Features Supported
+#### Features supported
```
Comment (KimberlyFields) [inline docs/platform/3_Authentication/6-provision-users-with-okta-scim.md:31]:
```suggestion
#### Supported features
```

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md [hunk 2]
```diff
@@ -173,25 +179,39 @@ Groups with the Harness app assignment now appear in **Groups**.
 
 You can edit or delete users from here.
 
-#### Group Push to Harness
+#### Group push to Harness
```
Comment (KimberlyFields) [inline docs/platform/3_Authentication/6-provision-users-with-okta-scim.md:182]:
```suggestion
#### Push assigned groups to Harness
```

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md [hunk 3]
```diff
@@ -173,25 +179,39 @@ Groups with the Harness app assignment now appear in **Groups**.
...
 ![](./static/provision-users-with-okta-scim-16.png)
+
 Click **Save**. You can see the status of this Push Group in your application.
 
 ![](./static/provision-users-with-okta-scim-17.png)
-If an error prevents adding, deleting, or updating an individual user to Harness, you must retry provisioning the user in Okta later, after resolving the issues. For more information, see [Troubleshooting Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-group-push-troubleshoot.htm).This group is now listed in your Harness account.
+
+
+:::note
+If an error prevents adding, deleting, or updating an individual user to Harness, you must retry provisioning the user in Okta later, after resolving the issues. For more information, see [Troubleshooting Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-group-push-troubleshoot.htm).
```
Comment (KimberlyFields) [inline docs/platform/3_Authentication/6-provision-users-with-okta-scim.md:200]: You have to reprovision a user in Okta to delete a user in Harness? That's how I read that sentence. Is that correct?

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md [hunk 4]
```diff
@@ -208,24 +228,42 @@ To update user attributes:
 3. Click the **Profile** tab, then click the **Edit** button.
 4. Update the desired attributes, then click **Save**.![](./static/provision-users-with-okta-scim-19.png)
 
-Only the five fields listed at the top of this section will be synced to Harness users. You can update values in other fields, but those values will be saved for this user only in Okta. They won't be reflected in Harness.  
-The Display name in Okta is displayed as the user name in Harness.
 
-#### Deactivate Users
+:::note
+Only the five fields listed at the top of this section will be synced to Harness users. You can update values in other fields, but those values will be saved for this user only in Okta. They won't be reflected in Harness.
```
Comment (KimberlyFields) [inline docs/platform/3_Authentication/6-provision-users-with-okta-scim.md:233]:
```suggestion
Only the five fields listed at the top of this section are synced to Harness users. You can update values in other fields, but those values are saved for this user in Okta only and are not updated in Harness.
```

File: docs/platform/3_Authentication/7provision-users-and-groups-with-one-login-scim.md [hunk 5]
```diff
@@ -204,9 +218,15 @@ Next, you create a rule in the Harness OneLogin app to create groups using the r
 9. Click **Save**.
 10. Click **Save** to save the app.
 
-If you have created users prior to adding the mapping rule, click Reapply Mappings in your Harness application User settings:Now that the app has a rule to set groups in Harness using the role you created, you can begin provisioning users using the app.
 
-#### Provision Users in Application
+:::note
+If you have created users prior to adding the mapping rule, click Reapply Mappings in your Harness application User settings.
```
Comment (KimberlyFields) [inline docs/platform/3_Authentication/7provision-users-and-groups-with-one-login-scim.md:223]:
```suggestion
If you have created users prior to adding the mapping rule, select **Reapply Mappings** in the **User** settings section of your Harness application.
```

File: docs/platform/3_Authentication/8-provision-users-and-groups-using-azure-ad-scim.md [hunk 6]
```diff
@@ -20,13 +20,13 @@ This topic explains how to configure Azure Active Directory (Azure AD) to automa
...
 
-### Review: Harness Azure AD SCIM Integration
+### Review: Harness Azure AD SCIM integration
 
 By using Azure AD as your identity provider, you can efficiently provision and manage users in your Harness Account, Org and Project. Harness' [SCIM](https://www.okta.com/blog/2017/01/what-is-scim/) integration enables Azure AD to serve as a single identity manager, for adding and removing users, and for provisioning User Groups. This is especially efficient for managing many users.
 
 In exchange for the convenience of Azure AD-provisioned users and groups, you must configure several aspects of Azure AD, as described in the following sections. You will also have restrictions on modifying Azure AD-provisioned users and groups natively within Harness, as described in [Limitations](#limitations).
 
-#### Features Supported
+#### Features supported
```
Comment (KimberlyFields) [inline docs/platform/3_Authentication/8-provision-users-and-groups-using-azure-ad-scim.md:29]:
```suggestion
#### Supported features
```

**Review by KimberlyFields — CHANGES_REQUESTED** [review]

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md (hunk 3 above)
Comment (rashminandasahoo) [reply docs/platform/3_Authentication/6-provision-users-with-okta-scim.md:200]: Updated to "If an error prevents adding, deleting, or updating an individual user to Harness, you must retry provisioning the user later, after resolving the issues. For more information, see [Troubleshooting Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-group-push-troubleshoot.htm)."

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md (hunk 1 above)
Comment (rashminandasahoo) [reply docs/platform/3_Authentication/6-provision-users-with-okta-scim.md]: Done

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md (hunk 2 above)
Comment (rashminandasahoo) [reply docs/platform/3_Authentication/6-provision-users-with-okta-scim.md]: Done

File: docs/platform/3_Authentication/6-provision-users-with-okta-scim.md (hunk 4 above)
Comment (rashminandasahoo) [reply docs/platform/3_Authentication/6-provision-users-with-okta-scim.md]: Done

File: docs/platform/3_Authentication/7provision-users-and-groups-with-one-login-scim.md (hunk 5 above)
Comment (rashminandasahoo) [reply docs/platform/3_Authentication/7provision-users-and-groups-with-one-login-scim.md]: Done

File: docs/platform/3_Authentication/8-provision-users-and-groups-using-azure-ad-scim.md (hunk 6 above)
Comment (rashminandasahoo) [reply docs/platform/3_Authentication/8-provision-users-and-groups-using-azure-ad-scim.md]: Done

**Review by SudheendraKatte — APPROVED** [review]
Approving the PR to unblock Rashmi.

**Pull request merged**